index.html

```diff
@@ -180,7 +180,35 @@ <h1>Skills</h1>
     </ul>
 
   </section>
-
+  <section class="contact-form">
+    <form class="my-form" action="https://formspree.io/f/mrgdbqpg" method="POST">
+      <p class="contact-form-content">I'm always interested in hearing about new projects, so if you'd like to chat
+        please get in touch.</p>
+      <input class="contact-form-name" type="text" name="name" placeholder="Full name">
```

Comment:
- [ ] Kindly make sure to add all validations as requested in the project requirements. Please add the `required` attribute which helps us to not submit empty form input. Also, add `maxlength="30"` validation to validate the maximum input character to be inserted in the name `input`. :wink: